src/amo/components/Addon/index.js

```diff
@@ -308,6 +311,15 @@ export class AddonBase extends React.Component {
             <h1 className="Addon-title" {...titleProps} />
             <p className="Addon-summary" {...summaryProps} />
 
+            <div className="Addon-badges">
+              {isFeatured ? (
+                <Badge
+                  type="featured"
+                  label={i18n.sprintf(i18n.gettext('Featured %(addonType)s'), { addonType })}
```

Comment:
Also, just in general, note that you can't really use placeholders in localizable strings. The only times we use it are for things like HTML (like anchor URLs).